app/components/polls/polls-edit/polls-edit-directive.js

```diff
@@ -49,6 +49,24 @@ angular.module('polls.edit.directive', [
         return '<bulbs-poll src="/poll/' + $scope.model.id + '/merged.json"></bulbs-poll>';
       };
 
+      $scope.validatePublication = function () {
+        $timeout(function () {
```

Comment:
Should leave a comment or figure out how to solve this correctly.

The `datetime-selection-modal-opener` interacts with the form in such a way that `modal-on-close="validatePublication()"` fires before the scope model data has changed.